.github/workflows/lint-markdown.yml

```diff
@@ -0,0 +1,25 @@
+name: Lint markdown
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened]
```

Comment:
Please could we filter this to only be on `docs/` paths? It's a quick workflow but sometimes getting an available runner can take time!